python/delta_sharing/delta_sharing.py

```diff
@@ -172,6 +180,25 @@ def load_as_spark(
         "`load_as_spark` requires running in a PySpark application."
     )
     df = spark.read.format("deltaSharing")
+    if delta_sharing_profile is not None:
```

Comment:
this block of code is duplicate same as below can we have a helper method for this?